query/src/storages/fuse/meta/v1/segment.rs

```diff
@@ -48,6 +48,7 @@ pub struct BlockMeta {
     pub file_size: u64,
     pub col_stats: HashMap<ColumnId, ColumnStatistics>,
     pub col_metas: HashMap<ColumnId, ColumnMeta>,
+    pub cluster_stats: HashMap<ColumnId, ColumnStatistics>,
```

Comment:
Just a reminder:

we need a `#[serde(default)]` attr for the new field `cluster_stats` to keep backward compatible in the final PR